### PR TITLE
DT-58 markups

### PR DESCRIPTION
**Pull request by BenRamchandani**

### CODE REVIEW

File: terraform/test/main.tf [hunk 1]
```diff
@@ -106,10 +106,21 @@ module "marklogic" {
   environment     = "test"
   vpc             = module.networking.vpc
   private_subnets = module.networking.ml_private_subnets
-  instance_type   = "t3.xlarge"
+  instance_type   = "t3.large"
```
Comment (BenRamchandani) [inline terraform/test/main.tf:109]: I've only done an instance refresh for one of the ASGs, but it seemed to work fine.

File: terraform/staging/outputs.tf [hunk 2]
```diff
@@ -55,10 +55,6 @@ output "vpc_id" {
   value = module.networking.vpc.id
 }
 
-output "gh_runner_ip" {
-  value = module.gh_runner.instance_ip
-}
```
Comment (BenRamchandani) [inline terraform/staging/outputs.tf:60]: Seems unnecessary, we can use gh-runner.vpc.local

File: terraform/test/main.tf (hunk 1 above)
Comment (BenRamchandani) [reply terraform/test/main.tf:109]: One thought from that: should we move the MarkLogic data EBS volumes into Terraform from CloudFormation? We really don't want to accidentally destroy them and Terraform gives us better controls for that.

Comment (hugh-emerson) [reply terraform/test/main.tf:109]: That's a good point. I'm surprised that the CF template doesn't have a deletion policy set.

**Review by hugh-emerson — APPROVED** [review]
LGTM

**Pull request merged**